pkg/volume/testing/testing.go

```diff
@@ -62,15 +63,16 @@ const (
 
 // fakeVolumeHost is useful for testing volume plugins.
 type fakeVolumeHost struct {
-	rootDir    string
-	kubeClient clientset.Interface
-	pluginMgr  VolumePluginMgr
-	cloud      cloudprovider.Interface
-	mounter    mount.Interface
-	exec       mount.Exec
-	nodeLabels map[string]string
-	nodeName   string
-	subpather  subpath.Interface
+	rootDir         string
+	kubeClient      clientset.Interface
+	pluginMgr       VolumePluginMgr
+	cloud           cloudprovider.Interface
+	mounter         mount.Interface
+	exec            mount.Exec
+	nodeLabels      map[string]string
+	nodeName        string
+	subpather       subpath.Interface
+	csiDriverLister *storagelisters.CSIDriverLister
```

Comment:
why is this a pointer? It is the only `VolumeHost` that saves a pointer to the interface instead of just using the interface (which will be a pointer to the struct anyway)

Reply:
You are right.  It doesn't have to be a pointer.